server/utils/bot/logger.go

```diff
@@ -109,12 +109,12 @@ func (bot *bot) logToAdmins(level, message string) {
 
 type NilLogger struct{}
 
-func (l *NilLogger) With(logContext LogContext) Logger         { return l }
-func (l *NilLogger) Timed() Logger                             { return l }
-func (l *NilLogger) Debugf(format string, args ...interface{}) {}
-func (l *NilLogger) Errorf(format string, args ...interface{}) {}
-func (l *NilLogger) Infof(format string, args ...interface{})  {}
-func (l *NilLogger) Warnf(format string, args ...interface{})  {}
+func (l *NilLogger) With(_ LogContext) Logger          { return l }
+func (l *NilLogger) Timed() Logger                     { return l }
+func (l *NilLogger) Debugf(_ string, _ ...interface{}) {}
+func (l *NilLogger) Errorf(_ string, _ ...interface{}) {}
+func (l *NilLogger) Infof(_ string, _ ...interface{})  {}
+func (l *NilLogger) Warnf(_ string, _ ...interface{})  {}
```

Comment:
Let's just remove the named argument.

Reply:
@hanzei This PR is done with dependabot. It seems it doesn't care much for your suggestions 😉

Reply:
@hanzei Are your suggestions blocking? Maybe @raghavaggarwal2308 can take a look or delegate this to someone

Reply:
I'm confused. Why does the Dependabot PR contain linter fixes? Aren't they in `master`?

Reply:
@hanzei I feel quite dumb. That lint fixing was done by a commit by myself above